examples/buffer_overflow_fuzz_test.cc

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// A fuzz target that causes an ASAN buffer overflow for a particular input.
+
+#include <cstdint>
+#include <cstddef>
+
+bool DoBufferOverflow(const uint8_t *Data, size_t Size) {
```

Comment:
I'm bringing this up once more again: Please pay attention to variable names and use the Google C++ variable naming convention (lower-case names, like `data` and `size`, here and everywhere else): https://google.github.io/styleguide/cppguide.html#Variable_Names